### ISSUE TITLE
Is there a Hybrid mode available?

### ISSUE THREAD

**Issue by huckphin** [facebook/hhvm#496]
Per this E-mail thread from James Michael DuPont, it was suggested to post the questions to forums as issues in GitHub:  https://groups.google.com/forum/?fromgroups#!topic/hiphop-php-dev/OsSxDnzStnY

I have a large PHP codebase that I want to start investigating using HipHop to compile with, but it is totally infeasible to correct all problems to justify the switch from PHP to HipHop.  I need to be able to show a small proof of concept that works by compiling only a small portion of the codebase.

I am looking for any suggestions or techniques that people have used to get started on moving large codebases (that are largely HipHop incompatible) over to HipHop.

One thought I had was this:  Is there was a way to create a binary file that would attempt the PHP transform into C++ and compile it?  If the transformation or the compilation fails, then one could bundle a PHP interpreter in the binary with the PHP file unmodified, and make the PHP interpreter communicate with the compiled source code.

So basically, I am trying to ask, what is everyone else doing to get the C++ performance for 10% of their code base, and how are you incrementally fixing the existing PHP codebase to be HipHop compatible?
